### PR TITLE
Streamline subsystem naming and content

### PR DESCRIPTION
**Pull request by edmondchuc**
Use more common/general terminology such as vocabularies, catalogs, etc instead of subsystem names (VocPrez, etc). 

This change is made to ensure Prez as a piece of software is more approachable to a wider range of users, including those that do not have a technical/semantic background.

Since the current implementation couples the key of the subsystem to the breadcrumbs labelling display, I've introduced a lookup function go from key to label called `getPrezSystemLabel`.

### CODE REVIEW

File: src/views/vocprez/VocPrezHomeView.vue [hunk 1]
```diff
@@ -18,35 +18,42 @@ const links = [
...
         </p>
         <p>
-            See the <RouterLink to="/v/vocab">Vocabularies Register</RouterLink> for a list of vocabularies this
-            instance of VocPrez knows about.
+            See the <RouterLink to="/v/vocab">Vocabularies Register</RouterLink> for a list of vocabularies.
         </p>
         <h2>Collections</h2>
         <p>
-            Collections are a grouping of terms (or Concepts), represented as a flat listing of Concepts.
+            A collectionis a flat grouping of terms (concepts).
```
Comment (jamiefeiss) [inline src/views/vocprez/VocPrezHomeView.vue:38]: Typo

**Review by jamiefeiss — COMMENTED** [review]
Looks good so far. It makes sense to have friendlier names for the subsystems, though I'm unsure if we should hide the original names completely. @nicholascar what do you think?

If we are hiding all appearances of CatPrez, etc. then there are several pages that are yet to be updated. I'm happy to list them if that's the case.

File: src/views/vocprez/VocPrezHomeView.vue (hunk 1 above)
Comment (edmondchuc) [reply src/views/vocprez/VocPrezHomeView.vue]: Thanks for the catch!

**Comment by nicholascar** [comment]
@nicholascar what do you think?

Fine by me. These are non-functional (display text, not code) changes and they can be whatever looks nice. 

If we have this wrong - perhaps we've used generic wording too much - we can always easily update again since it's all non-functional!

**Comment by edmondchuc** [comment]
> If we are hiding all appearances of CatPrez, etc. then there are several pages that are yet to be updated. I'm happy to list them if that's the case.

@jamiefeiss I think I've updated all the pages and content to remove the system names except for the about page. Happy to update the about page as well, but I thought maybe we should leave some hints of the system names there since the about pages details some of the technical implementations and capabilities of the system overall.

**Review by nicholascar — APPROVED** [review]

**Pull request merged**